src/app/cloud/[team-slug]/[project-slug]/(tabs)/settings/environment-variables/_components/Secret/index.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react'
```

Comment:
I would have rather seen this component colocated within the environments folder, but o need to change this now.